src/DeviceInterface/AndroidInterface.js

```diff
@@ -47,7 +47,12 @@ class AndroidInterface extends InterfacePrototype {
 
     postInit () {
         const cleanup = this.scanner.init()
-        this.addLogoutListener(cleanup)
+        this.addLogoutListener(() => {
+            cleanup()
+            if (this.globalConfig.isDDGDomain) {
+                notifyWebApp({ deviceSignedIn: {value: false} })
+            }
```

Comment:
This logic was extracted out of the `cleanup` function so that we can cleanup and reinitialise the scanner without telling the web app that it's signed out (when it may not be)